jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jetty-client/src/main/java/module-info.java

```diff
@@ -11,11 +11,15 @@
 // ========================================================================
 //
 
+import org.eclipse.jetty.ee10.websocket.client.config.JettyWebSocketClientConfiguration;
+
 module org.eclipse.jetty.ee10.websocket.jetty.client
 {
     requires org.eclipse.jetty.websocket.client;
 
-    requires static org.eclipse.jetty.ee10.webapp;
+    requires static transitive org.eclipse.jetty.ee10.webapp;
```

Comment:
I don't see the same surefire config in the pom.xml as for ee8 and ee9, and indeed I can't see any unit tests, where are they?

Reply:
This is needed for the `JakartaWebSocketShutdownContainer` which is really a fallback if the Jetty `JakartaWebSocketClientContainer` is used from a different servlet container. Otherwise if in Jetty it should just register as a bean on the `ContextHandler`.

Because of this it is hard to test in a unit test.